src/main/webapp/WEB-INF/tags/instructor/comments/pagination.tag

```diff
@@ -0,0 +1,14 @@
+<%@ tag description="Course Pagination" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<ul class="pagination">
+    <li><a href="${data.previousPageLink}">�</a></li>
```

Comment:
It's a « character (guillemet).
They are likely not parsed correctly in the diff view, but show up fine in the file.
They can be replaced with `&laquo;` and `&raquo;` too, though.